src/main/kotlin/com/sourcegraph/cody/edit/FixupService.kt

```diff
@@ -70,18 +71,28 @@ class FixupService(val project: Project) : Disposable {
       logger.warn("Edit code invoked when Cody not enabled")
       return false
     }
-    if (CodyStatusService.getCurrentStatus(project) == CodyStatus.CodyAgentNotRunning) {
-      runInEdt { CodyStartingNotification().notify(project) }
-      logger.warn("The agent is not connected")
-      return false
-    }
     if (!CodyEditorUtil.isEditorValidForAutocomplete(editor)) {
       runInEdt { EditingNotAvailableNotification().notify(project) }
       logger.warn("Edit code invoked when editing not available")
       getActiveSession()?.cancel()
       return false
     }
+
+    // TODO: We'll have to figure out a way to integration-test the ignore stuff.
+    // Right now during testing the policy comes back null, which depending on the
+    // account type might mean ignore-by-default. So we override it for now.
```

Comment:
let's have it somewhere in Linear

Reply:
I think we maybe should fix it right away.
I do not like mixing production and test code, and it would be useful to understand how it could be `null`. Possibly it's some kind of bug we are silently ignoring there.

Reply:
I think previously it might have been caused by a race with an agent startup.
After some re-arrangements of the testing logic problem seems to not appear anymore and whole custom logic here can be removed.